.circleci/config.yml

```diff
@@ -0,0 +1,41 @@
+version: 2
+jobs:
+  build:
+    docker:
+      - image: circleci/node:latest
```

Comment:
specific version is always better

Reply:
But, in this case, don't we want to make sure it is running all the time with the most modern node version so we can detect issues as soon as possible?

Besides of that, still does not run CI with that config, any idea why? :(